mobile/cmp/button/ExpandCollapseButton.js

```diff
@@ -28,15 +28,15 @@ export const [ExpandCollapseButton, expandCollapseButton] = hoistCmp.withFactory
 
         if (!gridModel?.treeMode) {
             console.error("No GridModel available with treeMode enabled. Provide via a 'gridModel' prop, or context.");
-            return button({icon: Icon.angleDown(), disabled: true, ...props});
         }
```

Comment:
This will now throw if there is no gridModel at all in context.  Previously the code was more gentle

Reply:
Good point! Reverted back to the safer logic